ci-operator/step-registry/openshift-qe/orion/cudn-density/openshift-qe-orion-cudn-density-commands.sh

```
@@ -0,0 +1,345 @@
+#!/bin/bash
+set -x
+
+if [ ${RUN_ORION} == false ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)